src/main/resources/static/js/multitool/PdfContainer.js

```diff
@@ -212,6 +214,48 @@ class PdfContainer {
     }
   }
 
+  async splitPDF(baseDocBytes, splitters) {
+    const baseDocument = await PDFLib.PDFDocument.load(baseDocBytes);
+    const pageNum = baseDocument.getPages().length;
+
+    splitters.sort(); // We'll sort the separator indexes just in case querySelectorAll does something funny.
```

Comment:
This is causing the empty pages and general wrong splitting.
```js
splitters.sort();
```
will convert the numbers to string and do a string compare, instead you would have to do
```js
splitters.sort((a, b) => a - b);
```
to have the correct order.

I would commit the change directly but seems like I don't have permissions (makes sense 😄)

Reply:
You know, I'm not even surprised. This is not the first time I made this mistake in JS. I'll fix it quickly. 

Thank you, fixed.